selfdrive/common/swaglog.cc

```diff
@@ -72,7 +73,7 @@ static void log(int levelnum, const char* filename, int lineno, const char* func
   char levelnum_c = levelnum;
   zmq_send(s.sock, (levelnum_c + log_s).c_str(), log_s.length() + 1, ZMQ_NOBLOCK);
 }
-static void cloudlog_common(int levelnum, bool is_timestamp, const char* filename, int lineno, const char* func,
+static void cloudlog_common(int levelnum, int frame_id, bool is_timestamp, const char* filename, int lineno, const char* func,
```

Comment:
this is becoming too specific, perhaps take a json object instead that just gets added